ci/tasks/build.sh

```diff
@@ -8,6 +8,7 @@ workspace_dir="$( cd ${release_dir} && cd ../../../.. && pwd )"
 
 export GOPATH=${workspace_dir}
 export PATH=${GOPATH}/bin:${PATH}
+export GOOS=${GOOS:-linux}
```

Comment:
To avoid dev confusion, let's make the default value consistent with what's in `build.yml`. Either:

 * not default it in either place (and making it required); or
 * also default `build.yml` to `linux` instead of `REPLACE_ME`

I tend to lean towards the latter.